plugin/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -20,4 +20,8 @@ grant {
     permission java.util.PropertyPermission "java.library.path", "read,write";
     permission java.util.PropertyPermission "*", "read,write";
     permission java.lang.RuntimePermission "shutdownHooks";
+
+    // Circuit Breaker
+    permission java.lang.RuntimePermission "getFileSystemAttributes";
+    permission java.io.FilePermission "/", "read";
```

Comment:
As we only check opensearch data folder, remove this permission?

Reply:
Good catch. Just removed this line and all tests pass. Pushed the change to the second commit.